resources/packs/msgraph/msgraphclient/adapter.go

```diff
@@ -3,6 +3,7 @@ package msgraphclient
 import (
 	nethttp "net/http"
 
+	"github.com/cockroachdb/errors"
```

Comment:
I think msgraph package is not in the correct spot here? The directory should only contain resource packs. Should this be moved into the ms365 provider? Seems like it is primary a client assigned with the provider

Reply:
Done, moved to the provider under its own module